### PR TITLE
Revamped console room Sounds

### PR DESCRIPTION
**Pull request by CommandrMoose**
Redid the preview screens and fixed up the names for the console structures.

### CODE REVIEW

File: src/main/java/net/tardis/mod/common/blocks/BlockGallifreyDirt.java [hunk 1]
```diff
@@ -22,24 +22,28 @@
 
 public class BlockGallifreyDirt extends Block implements IGrowable {
 
-    public BlockGallifreyDirt(Boolean isTop) {
+    public BlockGallifreyDirt(Boolean isTop, Boolean hasSnow) {
```
Comment (Suffril) [inline src/main/java/net/tardis/mod/common/blocks/BlockGallifreyDirt.java]: BlockStates, please

File: src/main/java/net/tardis/mod/common/dimensions/gallifrey/WorldProviderGallifrey.java [hunk 2]
```diff
@@ -34,11 +34,11 @@ public IRenderHandler getSkyRenderer() {
 		return SKY;
 	}
 	
-	@Nullable
-	@Override
-	public IRenderHandler getCloudRenderer() {
-		return new RenderVoid();
-	}
+	//@Nullable
```
Comment (Suffril) [inline src/main/java/net/tardis/mod/common/dimensions/gallifrey/WorldProviderGallifrey.java]: Just remove it entirely, don't bother with commenting this out if we're keeping clouds

File: src/main/java/net/tardis/mod/common/entities/EntityRaider.java [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+        this.getAttributeMap().registerAttribute(SPAWN_REINFORCEMENTS_CHANCE).setBaseValue(this.rand.nextDouble() * net.minecraftforge.common.ForgeModContainer.zombieSummonBaseChance);
+
+    }
+
+
+    @Override
+    protected void entityInit() {
+        super.entityInit();
+        this.getDataManager().register(SITTING, false);
+        this.getDataManager().register(TYPE, EntityRaider.EnumRaiderType.values()[rand.nextInt(EntityRaider.EnumRaiderType.values().length)].name());
```
Comment (Suffril) [inline src/main/java/net/tardis/mod/common/entities/EntityRaider.java]: Look at the changes made to companions and change to that for the random enum value

File: src/main/java/net/tardis/mod/common/entities/ai/EntityAIRaiderAttack.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
+package net.tardis.mod.common.entities.ai;
```
Comment (Suffril) [inline src/main/java/net/tardis/mod/common/entities/ai/EntityAIRaiderAttack.java]: Delete this entire class

File: src/main/java/net/tardis/mod/common/entities/controls/ControlFuel.java [hunk 5]
```diff
@@ -48,4 +52,14 @@ public void init(TileEntityTardis tardis) {
 				this.setSize(Helper.precentToPixels(1F), Helper.precentToPixels(1F));
 		}
 	}
+
+	@Override
+	public SoundEvent getUseSound() {
+		TileEntityTardis tardis = (TileEntityTardis) world.getTileEntity(this.getConsolePos());
+		if(tardis.isFueling()){controlSound = TSounds.control_refuel_start;}
+		else controlSound = TSounds.control_refuel_stop;
+		return controlSound;
```
Comment (Suffril) [inline src/main/java/net/tardis/mod/common/entities/controls/ControlFuel.java]: Can be simplified to:
```return tardis.isFueling() ? TSounds.control_refuel_start; : TSounds.control_refuel_stop;```

File: src/main/java/net/tardis/mod/common/entities/controls/ControlLandType.java [hunk 6]
```diff
@@ -48,4 +53,14 @@ public void init(TileEntityTardis tardis) {
 				this.setSize(Helper.precentToPixels(3F), Helper.precentToPixels(2F));
 		}
 	}
+
+	@Override
+	public SoundEvent getUseSound() {
+		TileEntityTardis tardis = (TileEntityTardis) world.getTileEntity(this.getConsolePos());
+		if(tardis.landOnSurface == true){controlSound = TSounds.control_landing_type_up;}
+		else controlSound = TSounds.control_landing_type_down;
+		return controlSound;
```
Comment (Suffril) [inline src/main/java/net/tardis/mod/common/entities/controls/ControlLandType.java]: Same simplification can apply here

File: src/main/java/net/tardis/mod/common/entities/controls/ControlStabilizers.java [hunk 7]
```diff
@@ -58,4 +67,15 @@ public void init(TileEntityTardis tardis) {
...
 
+	@Override
+	public SoundEvent getUseSound() {
+		TileEntityTardis tardis = (TileEntityTardis) world.getTileEntity(this.getConsolePos());
+		SystemStabilizers stab = tardis.getSystem(SystemStabilizers.class);
+		if(stab.isOn() == true) { controlSound = TSounds.control_stabilizers_on; }
+		else controlSound = TSounds.control_stabilizers_off;
+
+
+		return controlSound;
```
Comment (Suffril) [inline src/main/java/net/tardis/mod/common/entities/controls/ControlStabilizers.java]: Same simplification

File: src/main/java/net/tardis/mod/common/blocks/BlockGallifreyDirt.java (hunk 1 above)
Comment (CommandrMoose) [reply src/main/java/net/tardis/mod/common/blocks/BlockGallifreyDirt.java]: No idea how

Comment (Suffril) [reply src/main/java/net/tardis/mod/common/blocks/BlockGallifreyDirt.java]: I will sort this then, but you are fixing the rest

File: src/main/java/net/tardis/mod/common/entities/controls/ControlFuel.java [hunk 8]
```diff
@@ -12,6 +14,8 @@
 
 public class ControlFuel extends EntityControl {
 
+	public static SoundEvent controlSound;
```
Comment (LotuxPunk) [inline src/main/java/net/tardis/mod/common/entities/controls/ControlFuel.java]: Why don't set this one on the EntityControl class ? Heritage stuff,...

**Pull request merged**